app/models/user.rb

```diff
@@ -21,4 +22,22 @@ def User.digest(string)
       BCrypt::Engine.cost
     BCrypt::Password.create(string, cost: cost)
   end
+
+  def User.new_token
+    SecureRandom.urlsafe_base64
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
+  end
+
+  def authenticated? remember_token
+    return false if remember_digest.nil?
```

Comment:
change ` return false if remember_digest.nil?` to ` return unless remember_digest` or ` return false unless remember_digest`

Reply:
Noted it, thank you.